clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5843,10 +5843,15 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
       CmdArgs.push_back("-emit-llvm-uselists");
 
     if (IsUsingLTO) {
+      bool IsUsingOffloadNewDriver =
```

Comment:
I think these changes are from a different PR which was recently merged. Can you please sync with SYCL tip? Thanks